assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -196,6 +179,17 @@ function MediaPane(props) {
     Flags.INCREMENTAL_SEARCH_DEBOUNCE_MEDIA
   );
 
+  const isNullOrUndefined = (item) => item === null || item === undefined;
+  const activeItem = useMemo(
+    () =>
+      FILTERS.find(
+        (item) =>
+          !isNullOrUndefined(mediaType) &&
+          item.value.toString() === mediaType.toString()
+      ) ?? FILTERS[0],
+    [mediaType]
+  );
```

Comment:
Since the Dropdown component does this check already, can't you just pass `value={mediaType?.toString() ||  FILTERS[0]}` to it?

Reply:
Good catch. It works with `value={mediaType?.toString() || FILTERS[0].value}`. Changed.